Update for 1.21.5

rrvk opened the pull request:
Here a pull request so it works for 1.21.5

rrvk committed:
Update for 1.12.5

Thomas7520 commented:
Hey, sorry for being lazy. Thanks for your contribution. However I cannot build your own version of 1.21.5 for some obscure reasons. But it helped me :)

rrvk commented:
Aha i was using your mod and i updated my client to 1.21.5 but your mod did not support it yet so that is why i fixed it for my self and i thought i may as well give this back :)
